app/grandchallenge/algorithms/models.py

```diff
@@ -156,10 +156,23 @@ class Algorithm(UUIDModel, TitleSlugDescriptionModel):
     )
     use_flexible_inputs = models.BooleanField(default=True)
     repo_name = models.CharField(blank=True, max_length=512)
+    image_requires_gpu = models.BooleanField(default=False)
+    image_requires_memory_gb = models.PositiveIntegerField(default=4)
```

Comment:
Sorry, missed these before - can we default to `True` and `15`? Users always forget to set these which is why we changed the defaults in the form, which are not being populated from here.